<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Port - Login</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(42, 42, 42);
	color: #EAEAEA;
}

QLineEdit {
	padding: 10px;
	border: 1px solid #D96900;
	border-radius: 10px;
	selection-background-color: grey;
	selection-color: white;
}
QLineEdit:hover {
	border: 2px solid #EA7A10;
}
QLineEdit:focus {
	outline: none;
	border: 2px solid #FFAA00;

}
QPushButton {
	background: #D96900;
	border: 1px solid #D96900;
	border-radius: 10px;
}
QPushButton:hover {
	background: #EA7A10;
}
QPushButton:focus {
	background: #EA7A10;
}
QPushButton:pressed {
	background: #FFAA00;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next</family>
     <pointsize>96</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PORT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editEmail">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPassword">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSubmit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>SIGN IN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Ultimate Portfolio Manager</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	text-decoration: underline;
}</string>
   </property>
   <property name="text">
    <string>Don't have an account, Sign up now.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
